src/Components/AuthLogin/index.js

```diff
@@ -2,7 +2,52 @@ import React from "react";
 import "./styles.css";
 import { Link } from "react-router-dom";
 
+import { useAuth } from "../../Contexts/AuthContext";
+import { useState } from "react";
+import axios from "axios";
+import { useNavigate } from "react-router-dom";
+
 function AuthLogin() {
+  const navigate = useNavigate();
+
+  const [tempUserDetail, setTempUserDetail] = useState({
+    tempEmail: "",
+    tempPassword: "",
+  });
+
+  const handleChange = (e) => {
+    setTempUserDetail({
+      ...tempUserDetail,
+      [e.target.name]: e.target.value,
+    });
+  };
+
+  const { dispatchAuth } = useAuth();
+
+  const { tempEmail, tempPassword } = tempUserDetail;
+
+  const submitUserDetails = async () => {
+    try {
+      const response = await axios.post(`/api/auth/login`, {
+        email: tempEmail,
+        password: tempPassword,
+      });
+
+      console.log("ccccc", response);
```

Comment:
Remove console call

Reply:
Thanks for pointing it out i will remove it